microsite/pages/en/index.js

```diff
@@ -559,7 +559,7 @@ class Index extends React.Component {
               <a href="https://www.cncf.io">
                 Cloud Native Computing Foundation
               </a>{' '}
-              sandbox project
+              incubating project
```

Comment:
I think this should actually just be "project" now. Afaik incubating and graduated aren't separated in the same way as sandbox